leetcode/linked-list.md

```diff
@@ -0,0 +1,133 @@
+# linked-list
```

Comment:
# Invalid PULL Commits

** Invalid Commit Message: LEETCODE added Reverse Linked List **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE added Middle of the Linked List **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE added Palindrome Linked List **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE-1022 Merge Two Sorted Lists **
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE-1022 Remove Nth Node From End of List **
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']